timesketch/lib/analyzers/contrib/misp_analyzer_test.py

```diff
@@ -86,11 +92,18 @@ def test_attr_nomatch(self, mock_requests_post):
         message = analyzer.run()
         self.assertEqual(
             message,
-            ("MISP Match: 0"),
+            ("[filename] MISP Match: 0"),
         )
         mock_requests_post.assert_called_with(
             "https://test.com//attributes/restSearch/",
-            data={"returnFormat": "json", "value": "test.txt", "type": "filename"},
+            json={"returnFormat": "json", "value": ["test.txt"], "type": "filename"},
             headers={"Authorization": "test"},
             verify=False,
         )
+
+    @mock.patch("timesketch.lib.analyzers.interface.OpenSearchDataStore", MockDataStore)
+    def test_get_kwargs(self):
+        analyzer_init = misp_analyzer.MispAnalyzer("test_index", 1)
+        queries = analyzer_init.get_kwargs()
+        self.assertIsNotNone(queries)
+        self.assertGreaterEqual(len(queries), 4)
```

Comment:
Some minor changes/additions I would recommend:
* Checking for len >= 1 should be sufficient. Then you don't need to adjust the test in case you make the queries configurable or add hardcoded queries in the future.
* Also adding a check if all loaded kwargs have the correct keys would be a good idea.

Reply:
Maybe something like this:
```
required_keys = ["query_string", "attr", "timesketch_attr"]
for query in misp_queries:
    self.assertTrue(all(key in query for key in required_keys))
```